src/printer.ts

```diff
@@ -434,8 +441,15 @@ export class PugPrinter {
 						break;
 					}
 					default: {
-						nonPrefixAttributes += 1;
 						this.currentLineLength += tempToken.name.length;
+						if (hasNonPrefixAttributes) {
+							// This isn't the first non-prefix attribute, add a space and separator
+							if (this.tokenNeedsSeparator(tempToken)) {
+								this.currentLineLength += 2;
+							} else {
+								this.currentLineLength += 1;
```

Comment:
Could this be "simplified" to

```ts
this.currentLineLength += 1;
if (this.tokenNeedsSeparator(tempToken)) {
  this.currentLineLength += 1;
}
```

Or does this have a technical reason?

Reply:
Yes I actually had it like this first. But the part in my brain that used to write assembler code for Intel Pentium wanted to reduce the amount of CPU cycles per path, hence doing the above. 🤣 

Happy to change.

Reply:
I will let you decide ^^

Reply:
Changed already! 😉